### DISCUSSION TITLE
【投票】自定义书源功能

### DISCUSSION THREAD

**Discussion by freeok** [freeok/so-novel#68] [Polls]
1. 您是否希望能够自定义书源？
2. 对于书源，您有哪些建议？

**Comment by twgtang** [comment]
大佬能不能出一个聚合搜索功能，在一个书源收不到的情况下，不用频繁的切换书源

**Reply by freeok** [reply]
已经有计划了，请关注后续版本

**Comment by yjjj0007** [comment]
大佬，出一个精确查询功能，能查询所有书源。

**Comment by freeok** [comment]
> 大佬，出一个精确查询功能，能查询所有书源。

已经有了

**Comment by Wang-sptc** [comment]
大佬，能不能出个**faloo**书源

**Comment by Liunian06** [comment]
希望大佬能出一个兼容阅读app书源的功能
